templates/takeovers/_private_cloud_economics.html

```diff
@@ -0,0 +1,17 @@
+<section class="p-strip is-deep p-takeover--private-cloud-economics">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h1 class="p-takeover__title">Busting the myth of private cloud economics</h1>
+      <p class="p-takeover__text">Analyst firm 451 Research, highlights TCO savings using BootStack, Canonical&rsquo;s private managed cloud service. </p>
+      <p class="u-hide--medium u-hide--large">
+        <img class="p-takeover__image" src="{{ ASSET_SERVER_URL }}c54da84b-451-Research-NEG.png" alt="">
+      </p>
+      <p>
+        <a href="/engage/cloud-economics" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Busting the myth of private cloud economics', 'eventLabel' : 'Read the independent report', 'eventValue' : undefined });" class="p-button--neutral">Read the independent report</a>
```

Comment:
I assume this page is being built in a separate PR? Just causing Travis to fail

Reply:
Yeah it proposed.